src/coreclr/src/tools/Common/TypeSystem/Common/ParameterizedType.cs

```diff
@@ -30,5 +30,10 @@ public override TypeSystemContext Context
                 return _parameterType.Context;
             }
         }
+
+        public override void EnsureFullyLoadable()
+        {
+            _parameterType.EnsureFullyLoadable();
```

Comment:
Leaving this outside the core type system will also avoid is having to have a discussion about whether array of void or array of byrefs should be "loadable".